GUIDELINES_AND_INFO.md

```diff
@@ -8,4 +8,13 @@ This is
 - an API for Scratch account verification
 
 ## Guidelines
-None yet.
+(NOT DONE)
+Please read our [Code of Conduct](https://github.com/ScratchVerifier/ScratchVerifier/blob/develop/CODE_OF_CONDUCT.md) before proceeding. 
+The following behaviour is not encouraged:
```

Comment:
"not encouraged" is different from "discouraged".

Reply:
Gotta fix that.

Reply:
fixed